resources/assets/js/components/JobPreview.tsx

```diff
@@ -11,36 +17,112 @@ export interface JobPreviewProps {
   securityLevel: string;
-  /** Date the position begins */
-  startDate?: string;
   /** Government classification code for the position, i.e. CS */
   classification: string;
   /** Level for the classification, i.e. 03 */
   level: string;
+  /** How often are Flex Hours allowed */
+  flexHours: string;
+  // Description of Required Education
+  education: string;
+  // Telework is allowed
+  telework: string;
+  // Travel opportunities;
 }
 
-const JobPreview: React.FunctionComponent<JobPreviewProps> = ({
+const messages = defineMessages({
+  allowed: {
+    id: "jobPreview.remoteWork.allowed",
+    defaultMessage: "Allowed",
+    description: "Remote Work Permission.",
+  },
+  notAllowed: {
+    id: "jobPreview.remoteWork.notAllowed",
+    defaultMessage: "Not Allowed",
+    description: "Remote World Permission",
+  },
+  teleworkNever: {
+    id: "jobPreview.telework.never",
+    defaultMessage: "Never",
+    description: "Telework Frequency",
+  },
+  teleworkOccasionally: {
+    id: "jobPreview.telework.occasionally",
+    defaultMessage: "Occasionally",
+    description: "Telework Frequency",
+  },
+  teleworkSometimes: {
+    id: "jobPreview.telework.sometimes",
+    defaultMessage: "Sometimes",
+    description: "Telework Frequency",
+  },
+  teleworkFrequently: {
+    id: "jobPreview.telework.frequently",
+    defaultMessage: "Frequently",
+    description: "Telework Frequency",
+  },
+  teleworkAlways: {
+    id: "jobPreview.telework.always",
+    defaultMessage: "Always",
+    description: "Telework Frequency",
+  },
+  flexHoursNever: {
+    id: "jobPreview.flexHours.never",
+    defaultMessage: "Never",
+    description: "Flex Hours Frequency",
+  },
+  flexHoursOccasionally: {
+    id: "jobPreview.flexHours.occasionally",
+    defaultMessage: "Occasionally",
+    description: "Flex Hours Frequency",
+  },
+  flexHoursSometimes: {
+    id: "jobPreview.flexHours.sometimes",
+    defaultMessage: "Sometimes",
+    description: "Flex Hours Frequency",
+  },
+  flexHoursFrequently: {
+    id: "jobPreview.flexHours.frequently",
+    defaultMessage: "Frequently",
+    description: "Flex Hours Frequency",
+  },
+  flexHoursAlways: {
+    id: "jobPreview.flexHours.always",
+    defaultMessage: "Always",
+    description: "Flex Hours Frequency",
+  },
+  termLength: {
+    id: "jobPreview.termLength",
+    defaultMessage:
+      "{termMonths, plural, =0 {no months} one {# month} other {# months}}",
+    description: "Calculated term lenght in months",
+  },
+});
+
+const JobPreview: React.FunctionComponent<
+  JobPreviewProps & InjectedIntlProps
+> = ({
   title,
   department,
   city,
+  education,
   province,
+  flexHours,
   remoteWork,
-  salary,
   language,
   termLength,
   securityLevel,
-  startDate,
   classification,
+  telework,
   level,
-}: JobPreviewProps): React.ReactElement => {
+  intl,
+}: JobPreviewProps & InjectedIntlProps): React.ReactElement => {
   return (
     <div
```

Comment:
Looks like the majority of the markup has changed for this component, see
https://github.com/GCTC-NTGC/TalentCloud/blob/feature/job-builder/step-03-ui/resources/views/manager/builder-02.html.twig#L455-L521

Reply:
Yeah, I was trying to split some of the differences between the old design and the new design. It was a stupid move in hind sight. Fixed. 👍